docs/environment

```diff
@@ -1,28 +1,61 @@
-# Example environment file
+##
+## Example ~/.config/labwc/environment file.
+## Uncomment lines starting with one '#' to suit your needs.
+##
 
-# This allows xdg-desktop-portal-wlr to function (e.g. for screen-recording)
-XDG_CURRENT_DESKTOP=wlroots
+##
+## Use the XKB_DEFAULT_LAYOUT variable to set the keyboard layout. For example
+## to start with Swedish keyboard layout set it to 'se'. If you are unsure what
+## your country code is, refer to the layout section of:
+## /usr/share/X11/xkb/rules/evdev.lst
+##
+## Multiple keyboard layouts can be set by comma-separating the country codes.
+## If multiple layouts are used, specify the toggle-keybind using
+## XKB_DEFAULT_OPTIONS as show below.
+##
+## For further details, see xkeyboard-config(7)
+##
 
-# Set keyboard layout to Swedish
-XKB_DEFAULT_LAYOUT=se
+# XKB_DEFAULT_LAYOUT=se
+# XKB_DEFAULT_LAYOUT=se,de
+# XKB_DEFAULT_OPTIONS=grp:alt_shift_toggle
+# XKB_DEFAULT_OPTIONS=grp:shift_caps_toggle
 
-# Set two keyboard layouts and toggle between them using alt+shift
-XKB_DEFAULT_LAYOUT=se,de
-XKB_DEFAULT_OPTIONS=grp:alt_shift_toggle
+##
+## Force firefox to use wayland backend.
+##
 
-# Force firefox to use wayland backend
-MOZ_ENABLE_WAYLAND=1
+# MOZ_ENABLE_WAYLAND=1
 
-# Set cursor theme.
-# Find icons themes with the command below or similar:
-#     find /usr/share/icons/ -type d -name "cursors"
-XCURSOR_THEME=breeze_cursors
+##
+## Set cursor theme and size. Find system icons themes with:
+## `find /usr/share/icons/ -type d -name "cursors"`
+##
 
-# Disable hardware cursors. Most users wouldn't want to do this, but if you
-# are experiencing issues with disappearing cursors, this might fix it.
-WLR_NO_HARDWARE_CURSORS=1
+# XCURSOR_THEME=breeze_cursors
+# XCURSOR_THEME=capitaine-cursors
+# XCURSOR_SIZE=24
+
+##
+## Disable hardware cursors. Most users would not want to do this, but if you
+## are experiencing issues with disappearing cursors, this might fix it.
+##
+
+# WLR_NO_HARDWARE_CURSORS=1
+
+##
+## For Java applications such as JetBrains/Intellij Idea, set this variable
+## to avoid menus with incorrect offset and blank windows.
+## See https://github.com/swaywm/sway/issues/595
+##
+
+# _JAVA_AWT_WM_NONREPARENTING=1
+
+##
+## This allows xdg-desktop-portal-wlr to function (e.g. for screen-recording).
+## It is set automatically by labwc though, so it is only includeded here for
+## completeness.
+##
+
+# XDG_CURRENT_DESKTOP=wlroots
```

Comment:
AFAIR setting this prevents labwc to overwrite it. So it can also be used to force something else in here (this is just from memory though, didn't verify it).

Reply:
Correct.
Have amended comment to clarify this.